File: app/src/main/java/mx/edu/upqroo/kristenandroid/api/sie/containers/Materia.java
package mx.edu.upqroo.kristenandroid.api.sie.containers;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Materia {
    @SerializedName("acronimo")
    @Expose
    private String acronimo;
    @SerializedName("nombre")
    @Expose
    private String nombre;
    @SerializedName("profesor")
    @Expose
    private String profesor;
    @SerializedName("hora")
    @Expose
    private String hora;

    public String getAcronimo() {
        return acronimo;
    }

    public void setAcronimo(String acronimo) {
        this.acronimo = acronimo;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getProfesor() {
        return profesor;
    }

    public void setProfesor(String profesor) {
        this.profesor = profesor;
    }

    public String getHora() {
        return hora;
    }

    public void setHora(String hora) {
        this.hora = hora;
    }
}

File: app/src/main/java/mx/edu/upqroo/kristenandroid/api/sie/containers/Materia.kt
package mx.edu.upqroo.kristenandroid.api.sie.containers

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Materia {
    @SerializedName("acronimo")
    @Expose
    var acronimo: String? = null
    @SerializedName("nombre")
    @Expose
    var nombre: String? = null
    @SerializedName("profesor")
    @Expose
    var profesor: String? = null
    @SerializedName("hora")
    @Expose
    var hora: String? = null
}
